paper/benchmarkset.tex

```diff
@@ -745,6 +745,102 @@ \subsubsection{Computational challenges posed by the T4 lysozyme benchmarks}
 Accounting for water exchange into and out of these buried binding sites could also be a challenge in the polar binding site, as phenol binds with one ordered water~\cite{wei_model_2002}. However, to our knowledge, the simulation timescales for water sampling have not yet been examined, other than noting that water does not enter the polar binding site as ligands are removed~\cite{boyce_predicting_2009}. 
 In contrast, the apolar cavity is dry, both with and without a bound ligand, so water sampling is unlikely to be a challenge in this case.
 
+\subsection{Protein-ligand benchmarks: the first bromodomain of the BRD4 protein}
+\label{sec:t5}
+
+We extend our protein benchmarks to protein domains called bromodomains, which have recently been the topic of numerous studies in the search for high affinity ligands, using both 
+computational~\cite{Deepak:2017:Chem.Biol.Drug.Des., Dickson:2016:J.Chem.Phys., Raj:2016:J.Biomol.Struct.Dyn., Ran:2015:Mol.Biosyst., Zhao:2014:Bioorg.Med.Chem.Lett, Xu:2016:J.Med.Chem., Yang:2016:BioorganicChemistry} and experimental 
+structure-based techniques~\cite{ Zhang:2013:J.Med.Chem.,Bamborough:2015:J.Med.Chem., Ran:2015:J.Med.Chem., Gosmini:2014:J.Med.Chem., Chung:2011:J.Med.Chem., Xue:2016:J.Med.Chem., Fill:2012:Bioorg.Med.Chem., Fish:2012:J.Med.Chem.}. 
+In living cells, bromodomains are responsible for recognizing and binding to acetylated lysine residues present in the histone tails, a known epigenetic marker~\cite{Chung:2016:Prog.Med.Chem.}. They also bind a broad spectrum of small to 
```

Comment:
Small LaTeX-on-GitHub tip: Break lines at least every sentence (after every period), if not more often. That way, when we go in and make edits, it will be easy to see what was changed.